code/datums/status_effects/drug_effects.dm

```diff
@@ -58,3 +58,39 @@
 	name = "Seizure"
 	desc = "FJOIWEHUWQEFGYUWDGHUIWHUIDWEHUIFDUWGYSXQHUIODSDBNJKVBNKDML <--- this is you right now"
 	icon_state = "paralysis"
+
+/datum/status_effect/stoned
+	id = "stoned"
+	duration = 10 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/stoned
+	status_type = STATUS_EFFECT_REFRESH
+	var/original_eye_color
+
+/datum/status_effect/stoned/on_apply()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		original_eye_color = human_owner.eye_color
+		human_owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/cannabis) //slows you down
+		human_owner.eye_color = BLOODCULT_EYE //gives cultists plausible deniability
+		human_owner.update_body() //updates eye color
+		ADD_TRAIT(human_owner, TRAIT_BLOODSHOT_EYES, type) //dilates blood vessels in eyes
+		ADD_TRAIT(human_owner, TRAIT_CLUMSY, type) //impairs motor coordination
```

Comment:
recently the image of a chemist with a smoke machine incapacitating an entire sec team with weed smoke came to mind and i'm not sure if it's going to be a problem

Reply:
emergent gameplay mark as resolved 👍